### ISSUE TITLE
iOS7 Issue with the Slider setting

### ISSUE THREAD

**Issue by LittleOrangeC** [kgn/InAppSettings#28]
In InAppSettingsPSSliderSpecifierCell.m

It appears 

```
- (void)slideAction
```

 is not being called when the deployment target is iOS7.  I suspect it's something with the 

```
[self.valueSlider addTarget:self action:@selector(slideAction) forControlEvents:UIControlEventTouchUpInside+UIControlEventTouchUpOutside];
```

 line but I am still researching.

**Comment by kgn** [comment]
Awesome, thank you!

**Comment by LittleOrangeC** [comment]
Resolved with pull request #30
